### PR TITLE
Created TODO's throughout my code

### PR DESCRIPTION
**Pull request by KBrandtjen**
Created this for class 9 assignment

### CODE REVIEW

**Comment by KBrandtjen** [comment]
Overall, would like to organize my files better. I need to organize my CSS and divide it up between the multiple pages created. Right now I have most of it on the base.css page but want to break it up between all of my pages.  Create multiple pages based on what the code is doing. Have portfolio actually reflect my work that I have completed. Add a resume page to display my resume. Get rid of useless code.

File: css/base.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
+/* TODO There is so much I want to change witin the styling and look of this project. First of all I need to implement the hamburger menu. Next, I am designing from a mobile-first perspective so I want the hamburger menu and one project per section. I want to create a media query that will change for the desktop version of my portfolio. My desktop version will display 2-3 portfolios in a line. Also my hamburger menu should change into an inline nav bar. I also want to change my header to be more stylish. Since the content on my page will be portfolio projects and look fairly basic I want the header to be a place where I can showcase my design skills. My about me section is another area where I would like to showcase my CSS abilities. Also as I mentioned previously, I would like to add another section to my portfolio that is a personal resume. This will showcase my background but also my ability to create a resume online using web development skills. */
+
```
Comment (serashioda) [inline css/base.css:3]: I like the way you wrote out your TODO's in almost a story format, as if you are having a natural conversation with yourself in regards to what needs to be done, and what you would like to work towards. You've laid out the overview of what you deem as needing to be done/fixed, which will help you work on immediate action items, along with a great map that will keep your perspective as you complete your Portfolio. I will do my best to add to your already well-thought out TODO's.

Great job considering the end-design of mobile-first. As you wrote in my peer-review, hamburger menu for the nav bar for mobile-perspective makes a lot of sense. Keeping the user perspective in mind and how this needs to be laid out depending on user-perspective is key in having a successful outcome in your Portfolio, and you are already thinking that way which is setting you up for success. 

My suggestion is to list by importance so you can tackle the necessary first, which I can tell you already have in mind, but it's easy to view importance in priorities.  I know that seems obvious, and I can tell you have this mapped out already in your mind from the way you wrote out the above, but having it listed in order of importance allows you to have a flexible map of TODO's without constantly having to remap them in your head (1 less thing to have to constantly think about! :) ), and tackle your TODO's methodically, while allowing you to be flexible as you have to adapt to things that come up.  That helps me, but you know what works for you, and I think you have great points laid out!

File: index.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <!-- TODO: Wihtin the html page I need to get rid of commented code or get that code to work. Most of the commented code wants deals with the category and title filters. I had this working previously but since adding new things they are not working. I either need to get them working or remove them all together. They are making my code hard to read.  -->
```
Comment (serashioda) [inline index.html:4]: As you commented yourself, getting rid of corpse code will help you see what is going on in your code better and see what isn't working/necessary. I had a lot of that prior to class_09 branch, and I still have some, but some were parts of old code from prior to refactoring that wasn't going to be used for or adapted to anything. getting rid of these helped me really clean up code.

File: index.html [hunk 3]
```diff
@@ -94,6 +97,7 @@ <h2 class="authorClass">Author</h2>
 <footer>
   <section id="description-total">
     <dt> Total Words in Project Descriptions:</dt> <dd class="words"></dd>
+    <!-- Right now I have a number counter that is counting the number of words in the project descriptions. This was for functionality purposes but I would like to incorporate some stats that are meaningful to my site. Since my portfolio is a record of all the things I have worked on I think that I would like to track something more relateable. I was thinking about adding a category to my page that was either hours spent on each project or skills used. My stats could then be total # of skills used or total # of hrs spent on all projects. -->
```
Comment (serashioda) [inline index.html:100]: Great job having in mind how to adapt the counter. I think a lot of us weren't quite sure how to adapt it to our Portfolio at first, and first incorporated it in a way similar to the lab. I like your idea about using this functionality for time spent, or counting the skills per project to showcase your abilities that you used for that project.

File: js/articleView.js [hunk 4]
```diff
@@ -3,6 +3,8 @@
 (function(module) {
   var articleView = {};
 
+// TODO I need to clean up this code and get rid of the commented code or figure out how to use it. Again, this code relates to the category filters. I would like to get them working. Even though at this point they seem a little useless. As I add more projects to my portfolio I think they will become much more useful.
+
```
Comment (serashioda) [inline js/articleView.js:7]: Getting rid of corpse code will really help you see your code and debug, and add more functionality. You already noted this, so you know what to do there! I suggest commenting throughout so you can really appreciate what your code is doing ( less to think about!), and add to and refactor.

File: js/articleView.js [hunk 5]
```diff
@@ -61,6 +63,8 @@
         $('#title-filter').append(a.toHtml('#title-filter-template'));
       };
     });
+    // TODO I wanted to add this to the JSON page but couldn't comment. I want to change the information in my projects section. I want them to actually reflect the projects I have created so far in my coding. RIght now I have photos and some filler information but I would like it to actually reflect the projects I have worked on.
```
Comment (serashioda) [inline js/articleView.js:66]: As you've mentioned and know already, I think we are at the point in our portfolio where finding that relevant data, what material you want to present, etc, will be important for the Portfolio to take shape now that we have a lot of the functionality. I have to go through this process still myself. I like the idea you mentioned previously about using the counter to count/show the skills used in each project, or time spent to finish the project.

File: js/statsView.js [hunk 6]
```diff
@@ -4,3 +4,5 @@ articleView.renderFooter = function() {
 };
 
 Project.fetchAll(articleView.renderFooter);
+
```
Comment (serashioda) [inline js/statsView.js:7]: Great job repeating and clarifying components of TODO's that are relevant to other parts of your code/files.

**Pull request merged**